src/aerospike_clj/client.clj

```diff
@@ -153,6 +161,11 @@
         ^Integer (.generation ^Record record)
         ^Integer (.expiration ^Record record)))))
 
+(defn- batch-read->map [^BatchRead br]
+  (assoc (record->map (.record br))
+         :index
+         (.toString (.userKey (.key br)))))
```

Comment:
You should check whether one of these properties may be null
It's usually safer to use String.valueOf()

Reply:
This is not necessary since those objects cannot be created with null values.